config/routes.rb

```diff
@@ -4,14 +4,23 @@
   # get 'sessions/destroy'
   # get 'users/new'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  
+  resources :users do
```

Comment:
This line gives me error message saying, "unexpected end of input".
You can remove the "do" for **line 7 in routes.rb**
![image](https://user-images.githubusercontent.com/35232348/95849265-cf9c4780-0d81-11eb-9735-e235e856b5a8.png)